src/clndr.js

```diff
@@ -205,10 +211,11 @@
                 if (this.options.lengthOfTime.startDate) {
                     this.intervalStart =
                         moment(this.options.lengthOfTime.startDate)
-                            .startOf('day');
+                            .startOf('week');
```

Comment:
I get that a 3 day calendar has questionable usefulness, but this problem would extend to a calendar that only shows 1 day at a time. That feels like a more useful example. If we can support it we should try!